blueman/main/NetworkManager.py

```diff
@@ -19,9 +21,10 @@ class NMConnectionError(Exception):
 class NMConnectionBase:
     conntype: str
 
-    def __init__(self, service, reply_handler=None, error_handler=None):
+    def __init__(self, service: Service, reply_handler: Callable[[], None],
+                 error_handler: Callable[[Union[NMConnectionError, GLib.Error]], None]):
```

Comment:
This is not just adding typing, a callback is now mandatory. It should be a separate commit explaining why.

Reply:
Split out. Sorry for the mess. I was too happy when I finally figured things out. :sweat_smile:

Reply:
No worries :+1: